src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/services/RetryApiService.kt

```diff
@@ -32,7 +33,8 @@ class RetryApiService(
     // Timeout for NO_RESPONSE is wrapped in a WebClientRequestException
     it is ReadTimeoutException || it.cause is ReadTimeoutException ||
       it is ConnectTimeoutException || it.cause is ConnectTimeoutException ||
-      it is SocketException || it.cause is SocketException
+      it is SocketException || it.cause is SocketException ||
+      it is PrematureCloseException || it.cause is PrematureCloseException
```

Comment:
PrematureCloseException appears to be thrown when the network 'connection is closed' error occurs